src/ResourceManager/Compute/Commands.Compute/Extension/DSC/SetAzureVMDscExtensionCommand.cs

```diff
@@ -460,20 +460,6 @@ private ConfigurationUris UploadConfigurationDataToBlob()
                     configurationDataBlobReference.Uri.AbsoluteUri,
                     () =>
                     {
-                        if (!Force && configurationDataBlobReference.Exists())
```

Comment:
@eshaparmar This is just refactoring the code to have the same behavior using the new ConfirmAction overload.  the new code, after the ShouldProcess check, will show the ShouldContinue prompt unless the user has specified the Force parameter, when the configurationDataBlobReference exists.